va/validation-authority.go

```diff
@@ -257,6 +348,7 @@ func (va ValidationAuthorityImpl) validate(authz core.Authorization, challengeIn
 
 		logEvent.Challenge = authz.Challenges[challengeIndex]
 		if err != nil {
+			fmt.Println(nil, err, err == nil)
```

Comment:
Whoops! It is. I didn't understand how reflection works in Go and it was causing issues. I'll remove it.